src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -311,14 +316,10 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
                 </>
               ) : (
                 <>
-                  {pastAuctionResults.length > 0 && (
+                  {results.length > 0 && (
```

Comment:
Fixing this one because it was rendering the wrong auction results (only when feature flag is disabled).